treeherder/model/derived/jobs.py

```diff
@@ -2093,6 +2097,12 @@ def store_performance_series(
 
             db_series_json = performance_series[0]['blob']
 
```

Comment:
I'd probably rewrite this as:
    # new blobs are gzip'ed to save space, old ones may not be
    try:
        db_series_json = zlib.decompress(performance_series[0]['blob'])
    except zlib.error:
        db_series_json = performace_series[0]['blob']